api/src/main/java/org/openmrs/util/OpenmrsClassLoader.java

```diff
@@ -358,25 +356,6 @@ public static void destroyInstance() {
 		
 		log.info("this classloader hashcode: {}", OpenmrsClassLoaderHolder.INSTANCE.hashCode());
 		
-		//Shut down and remove all cache managers.
-		List<CacheManager> knownCacheManagers = CacheManager.ALL_CACHE_MANAGERS;
-		while (!knownCacheManagers.isEmpty()) {
-			CacheManager cacheManager = CacheManager.ALL_CACHE_MANAGERS.get(0);
-			try {
-				//This shuts down and removes the cache manager.
-				cacheManager.shutdown();
-				
-				//Just in case the the timer does not stop, set the cacheManager 
-				//timer to null because it references this class loader.
-				Field field = cacheManager.getClass().getDeclaredField("cacheManagerTimer");
-				field.setAccessible(true);
-				field.set(cacheManager, null);
-			}
-			catch (Exception ex) {
-				log.error(ex.getMessage(), ex);
-			}
-		}
-		
```

Comment:
Infinispan caches should be properly shutdown by Spring and Hibernate shutdown methods...